docs/sources/flow/reference/components/otelcol.connector.spanmetrics.md

```diff
@@ -259,42 +274,490 @@ otelcol.exporter.otlp "production" {
 
 ### Sending metrics via a Prometheus remote write
 
-In order for a `target_info` metric to be generated, the incoming spans resource scope
-attributes must contain `service.name` and `service.instance.id` attributes.
+The generated metrics can be sent to a Prometheus-compatible database such as Grafana Mimir.
+However, extra steps are required in order to make sure all metric samples are received.
+
+`otelcol.connector.spanmetrics` is an OTLP-native component. As such, it aims to preserve the resource attributes of spans.
+
+1. For example, let's assume that there are two incoming resources spans with the same `service.name` and `k8s.pod.name` resource attributes.
+   {{< collapse title="Example JSON of two incoming spans." >}}
```

Comment:
This PR is notable for using these `collapse` features inside the Examples section of Flow component docs. AFAIK we haven't done this before. I think it could really help illustrate this issue to people who are not familiar with OTLP.

<img width="807" alt="Screenshot 2024-01-05 at 18 00 45" src="https://github.com/grafana/agent/assets/5834788/07379904-61f4-46f0-9b92-594820dcf57a">